### PR TITLE
Feature/1406 ids on error msg

### LINKED ISSUE #1406: As an admin, I'd like error messages to include the user and org ids
For those admins that manage more than one org, it is useful to know which org the error is coming from.

We can have a message similar to the one Salesforce adds at the beginning of an error message:

Apex script unhandled exception by user/organization: xxxxxxxxxxxxxxxx/xxxxxxxxxxxxx

### PR DESCRIPTION
**Pull request by ceiroa**
# Warning
# Info
# Issues
Fixes #1406.

### CODE REVIEW

**Comment by ceiroa** [comment]
@njjc ~ ready for review.

File: ApexDocContent/Allocations.htm [hunk 1]
```diff
@@ -7,8 +7,7 @@
 			<li>Manage Allocations page</li>
 			<li>Allocations rollups</li>
 			<li>Make Default Allocations feature</li>
-			<li>Multi currency support
-			<li>
+			<li>Multi currency support<li>
```
Comment (njjc) [inline ApexDocContent/Allocations.htm]: The closing tag isn't /closing

File: src/classes/ERR_Notifier.cls [hunk 2]
```diff
@@ -146,7 +146,8 @@ public with sharing class ERR_Notifier {
         mail.setReplyTo('donotreply@salesforcefoundation.org');
         mail.setSenderDisplayName('Nonprofit Starter Pack');
         mail.setSubject('NPSP Error');
-        string bodyString = '';
+        string bodyString = 'Organization: ' + UserInfo.getOrganizationName() + '(' + UserInfo.getOrganizationId() + ')';
+        bodyString += 'User: ' + UserInfo.getUserName() + '(' + UserInfo.getUserId() + ')';
```
Comment (njjc) [inline src/classes/ERR_Notifier.cls]: Do you have an example of what this looks like? Is this on a single line with no space between the ) and User:?

Comment (njjc) [inline src/classes/ERR_Notifier.cls]: There are \n later on, we might need one here.

**Comment by ceiroa** [comment]
Ok, I added the / for the closing tag, and a new line after the one with org info. Let me know how this looks.

**Pull request merged**